opslevel/resource_opslevel_property_assignment.go

```diff
@@ -106,11 +106,12 @@ func (resource *PropertyAssignmentResource) Create(ctx context.Context, req reso
 
 	definition := planModel.Definition.ValueString()
 	owner := planModel.Owner.ValueString()
-	value := opslevel.JsonString(planModel.Value.ValueString())
 	input := opslevel.PropertyInput{
 		Definition: *opslevel.NewIdentifier(planModel.Definition.ValueString()),
 		Owner:      *opslevel.NewIdentifier(planModel.Owner.ValueString()),
-		Value:      value,
+	}
+	if planModel.Value.IsNull() {
+		input.Value = opslevel.JsonString(planModel.Value.ValueString())
```

Comment:
```suggestion
		input.Value = opslevel.JsonString("")
```
Totally optional, but would this be clearer?

Reply:
🤦 the null check is suppose to be "no null". 🤦